server/routes/shared_api/autocomplete/helpers.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from typing import List
+from urllib.parse import urlencode
+
+from flask import current_app
+import requests
+
+MAPS_API_URL = "https://maps.googleapis.com/maps/api/place/autocomplete/json?"
+MIN_CHARACTERS_PER_QUERY = 3
+MAX_NUM_OF_QUERIES = 4
+RESPONSE_COUNT_LIMIT = 5
+
+
+def find_queries(user_query: str):
```

Comment:
Nit: add typing `def find_queries(user_query: str) -> List[str]:`

(and to other functions below)